model/build.go

```diff
@@ -149,3 +154,32 @@ func RemoveBuild(buildID string) error {
 
 	return errors.Wrap(db.C(BuildsCollection).RemoveId(buildID), "deleting build record")
 }
+
+const (
+	builderFieldNum  = 1
+	buildNumFieldNum = 2
+)
+
+func makeBinaryRepresentation(builder string, buildNum int) []byte {
+	var buf bytes.Buffer
+	builderBytes := []byte(builder)
+	buildNumBytes := []byte(strconv.Itoa(buildNum))
+	binary.Write(&buf, binary.BigEndian, uint32(builderFieldNum))
```

Comment:
Why doesn't it work to use the buffer's Write method for this?

Reply:
Because this is writing the integer as its binary representation (in BigEndian, because I had to pick one and that's what network stuff uses), and from what I can tell using the "binary" methods is the only way to get a byte array binary representation out of an int. 

See https://stackoverflow.com/questions/16888357/convert-an-integer-to-a-byte-array for example.

I don't actually know if we want to do it this way but I figured I'd write the code and see what you want in case it's too weird.

Reply:
Oh I get it. The Write method writes a slice of bytes